Спринт 13 на проверку

PavelRomashin opened the pull request:
Спринт 13 на проверку

PavelRomashin committed:
Спринт 13 на проверку

SeveNChaK commented on app/src/main/java/com/example/playlistmaker/PlayerActivity.kt:
```diff
@@ -0,0 +1,91 @@
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_player)
+        val backButton = findViewById<Button>(R.id.backButton)
+        val TrackImage = findViewById<ImageView>(R.id.TrackImage)
```
⏫ Обычно с маленькой буквы переменные называются, почему-то у тебя часть так, а часть по другому

SeveNChaK commented on app/src/main/java/com/example/playlistmaker/PlayerActivity.kt:
```diff
@@ -0,0 +1,91 @@
+        TrackLength.text = SimpleDateFormat("mm:ss", Locale.getDefault()).format(track.trackTime)
+        Glide.with(TrackImage).load(track.getCoverArtwork())
+            .placeholder(R.drawable.placeholder_player).centerCrop()
+            .transform(RoundedCorners(resources.getDimensionPixelSize(R.dimen.track_icon_round_size)))
```
⚠️ Ты используешь ресурс для треко в списке (там значение 2), а в плеере скругление больше. В макете значение 8

SeveNChaK commented on app/src/main/java/com/example/playlistmaker/SearchActivity.kt:
```diff
@@ -150,6 +156,10 @@ class SearchActivity : AppCompatActivity() {
             searchHistory.write(trackLibraryHistory)
             trackAdapterHistory.notifyDataSetChanged()
+
+            val playerIntent = Intent(this, PlayerActivity::class.java)
```
⏫ Строки 143-145 идентичны строкам 160-162. Я бы вынес их в метод и передавал туда нужный трек

SeveNChaK commented on app/src/main/res/layout/activity_player.xml:
```diff
@@ -0,0 +1,342 @@
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
```
⏫ Когда вью лежит внутри вертикальной `ScrollView` то высота должно быть `wrap_content`

PavelRomashin committed:
Спринт 13 исправление замечаний:
1) В классе PlayerActivity.kt исправлены названия переменных. Теперь с маленькой буквы названия.
2) Радиус скругления обложки исправлен на 8.
3) Вызов PlayerActivity.kt вынесен в отдельную функцию в SearchActivity.kt.
4) Высота в scrollView теперь имеет параметр wrap_content.
